Cesium.CodeGen/Ir/Expressions/SubscriptingExpression.cs

```diff
@@ -35,7 +35,12 @@ public IExpression Lower()
     public IValue Resolve(IDeclarationScope scope)
     {
         if (_expression is not IdentifierExpression identifier)
-            throw new WipException(230, "Subscription supported only for IdentifierConstantExpression");
+        {
+            if (_expression is not IValueExpression IValueExpresssion)
+                throw new CompilationException($"{_expression} is not IValueExpression");
+
+            return IValueExpresssion.Resolve(scope);

```

Comment:
Please add a new test verifying this functionality. Take a look at [the testing guide](https://github.com/ForNeVeR/Cesium/blob/0b8a10d10c898c190463a43eb3053913d9f4ca63/docs/tests.md) and let me know if it's unclear how to proceed.

Reply:
Ok! For now everything looks clear.